layouts/partials/top-nav.html

```diff
@@ -1,4 +1,4 @@
-<div  class="nav top-nav">
+<div  class="top-nav">
```

Comment:
I understand that this is named `top-nav` because it's at the top of the *section*, but I think this name could cause some confusion. I would assume that this means the main navigation (which is at the very top of the window).

Reply:
Resolved in #94